docs/reference/configuration-reference/general-settings.md

```diff
@@ -268,6 +268,9 @@ $$$tilemap-url$$$ `map.tilemap.url` ![logo cloud](https://doc-icons.s3.us-east-2
 `migrations.retryAttempts`
 :   The number of times migrations retry temporary failures, such as a network timeout, 503 status code, or `snapshot_in_progress_exception`. When upgrade migrations frequently fail after exhausting all retry attempts with a message such as `Unable to complete the [...] step after 15 attempts, terminating.`, increase the setting value. **Default: `15`**
 
+`migrations.useCummulativeLogger`
```

Comment:
````suggestion
`migrations.useCummulativeLogger` {applies_to}`stack: ga 9.2`
````
Is this from 9.2? If not, you can adjust my suggestion, or ignore it if it's valid since 9.0 included